account_customer_wallet/models/res_partner.py

```diff
@@ -71,3 +73,23 @@ def _compute_customer_wallet_balance(self):
             partner.customer_wallet_balance = sum(
                 -total["total"] for total in totals if total["partner_id"] in child_ids
             )
+
+    def _search_customer_wallet_balance(self, operator, value):
+        # This is a complete and utter hack. Don't do what I did.
+        if operator in ("=", "!=", ">", ">=", "<", "<=", "=?", "in", "not in"):
+            if operator in ("=", "=?"):
+                operator = "=="
+            filter_string = "partner.customer_wallet_balance {operator} {value}".format(
+                operator=operator, value=value
+            )
+            filtered = self.search([]).filtered(
+                lambda partner: safe_eval(
+                    filter_string,
+                    {"partner": partner},
+                )
+            )
```

Comment:
Wouldn't it be faster (although more risky?) to it with an SQL query ?

Reply:
That would require me to be better at writing SQL than I am ;)

But yes, you're right.